doc/python/cable_cell.rst

```diff
@@ -93,4 +93,147 @@ Cable cells
 
     properties of an ionic species.
 
+.. _pycableprobes:
+
+Cable cell probes
+-----------------
+
+Cable cell probe addresses are definied analagously to their counterparts in
+the C++ API (see :ref:`cable-cell-probes` for details). Sample data recorded
+by the Arbor simulation object is returned in the form of a NumPy array,
+with the first column holding sample times, and subsequent columns holding
+the corresponding scalar- or vector-valued sample.
+
+Location expressions will be realised as zero or more specific sites on
+a cell; probe addresses defined over location expressions will describe zero,
+one, or more probes, one per site.
+
+Each of the functions described below generates an opaque :class:`probe`
+object for use in the recipe :py:func:`get_probes` method.
+
+Membrane voltage
+   .. py:function:: cable_probe_membrane_voltage(locations)
+
+   Cell membrane potential (mV) at the sites specified by the location
+   expression string ``locations``. This value is spatially interpolated.
+
+   Metadata: the explicit :class:`location` of the sample site.
+
+   .. py:function:: cable_probe_membrane_voltage_cell()
+
+   Cell membrane potential (mV) associated with each cable in each CV of
+   the cell discretization.
+
+   Metadata: the list of corresponding :class:`cable` objects.
```

Comment:
On the second point first, there more detailed explanation about how to capture sampled data is in the `simulation` documentation. If that bit is clear, I can add a link from here to there. There's also information there about what probe metadata is meant to be.

Could you take a look at that bit and see if it helps sufficiently?

Reply:
I thought the python 'simulation' docs were actually really helpful, and my confusion was more related to the cpp docs and the concept docs. It would be nice to have something that describes the full probing/sampling procedure and links to the correct locations in the docs. 
I will think a bit more about this and discuss offline. For the scope of this PR, the newly added changes are really good. So I think we can resolve this issue.